kube_proxy/requirements.txt

```diff
@@ -0,0 +1 @@
+# integration pip requirements
```

Comment:
can you add a `requirements.in` file as well? Since it'll be empty it's enough to just have it here.